plugins/rtmp-services/rtmp-common.c

```diff
@@ -837,12 +837,6 @@ static const char *rtmp_common_key(void *data)
 	return service->key;
 }
 
-static bool supports_multitrack(void *data)
```

Comment:
This function was added by 189fc7ab6a. But later commit af090573952eb0b removed the callback `supports_multitrack`.